rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -849,4 +832,55 @@ public void setLraService(LRAService lraService) {
     public URI getEndNotificationUri() {
         return afterURI;
     }
+
+    private int forget(Client client) {
+        if (forgetURI == null) {
+            return TwoPhaseOutcome.FINISH_OK;
+        }
+
+        boolean close;
+
+        if (client != null) {
+            close = false;
+        } else {
+            close = true;
+            client = ClientBuilder.newClient();
+        }
+
+        LRALogger.logger.debugf("LRARecord.forget calling DELETE on %s with parent: %s",
+                    forgetURI, parentId == null ? "null" : parentId);
+
+        try {
+            // let the participant know he can clean up
+            WebTarget target2 = client.target(forgetURI);
```

Comment:
we can now rename the variables to drop the suffix index

Reply:
+1